app/models/user.rb

```diff
@@ -1,2 +1,3 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] You didn't implement the custom methods for any of the models. Please make sure to implement all custom methods as requested in the project requirements.

![image](https://github.com/evansnyamekye/Blog-application/assets/83803414/f0999c98-e19a-4c79-87a7-dc2d221bb975)

Reply:
Hi, @thecodechaser Fixed all required changes as requested. 

Thank you 😺.